src/lib/pci_virtio_net_vmnet.c

```diff
@@ -139,7 +139,7 @@ struct virtio_net_rxhdr {
  * Debug printf
  */
 static int pci_vtnet_debug;
-#define DPRINTF(params) if (pci_vtnet_debug) printf params
+#define DPRINTF(params) if (pci_vtnet_debug) log_info params
```

Comment:
there shouldn't be any need to change this debug code. this won't be enabled for release builds and there should be no need to redirect to a logger.